docker/base-deps/Dockerfile

```diff
@@ -48,16 +46,21 @@ RUN sudo apt-get update -y && sudo apt-get upgrade -y \
         netbase \
         openssh-client \
         gnupg; fi) \
-    && wget \
-        --quiet "https://repo.anaconda.com/miniconda/Miniconda3-py37_23.1.0-1-Linux-${HOSTTYPE}.sh" \
+    && if [ "$PYTHON_VERSION" = "3.11" ]; then \
+            MINICONDA_VERSION="py310_23.3.1-0"; \
+            LIBGCC="libgcc-ng"; \
+        else \
+            MINICONDA_VERSION="py37_23.1.0-1"; \
+            LIBGCC="libgcc-ng libffi=3.3"; \
```

Comment:
Could we add a TODO for the related libffi=3.3 patch? I remember there's a link to an issue on conda.

Reply:
https://github.com/ray-project/ray/issues/33299

Reply:
Totally!